ops/dev-deploy.tmpl.yaml

```diff
@@ -280,7 +283,14 @@ extraDeploy:
                   - secretRef:
                       name: {{ template "hyrax.fullname" . }}
                 env:
-                  {{- toYaml .Values.workerAuxiliary.extraEnvVars | nindent 12 }}
+                  {{- range .Values.workerAuxiliary.extraDeployEnvVars }}
+                  - name: {{ .name }}
+                    value: {{ .value | quote }}
+                  {{- end }}
+                  {{- range .Values.workerAuxiliary.extraEnvVars }}
+                  - name: {{ .name }}
+                    value: {{ .value | quote }}
+                  {{- end }}
```

Comment:
Ooo, this looks interesting, did this work?

Reply:
yes!

Reply:
I'm assuming it still gave us both versions of the `GOOD_JOB_QUEUES` variable though?

Reply:
> I'm assuming it still gave us both versions of the GOOD_JOB_QUEUES variable though?

Actually, I'm only seeing one `GOOD_JOB_QUEUES` variable per worker deployment! 🎉 Though they both have the same value (`"*"`), so not 100% there it seems. But still, this looks promising!

Reply:
So, finding out a few things here, this does work, but it works just the same as the previous code did. I have ran test over test with the output of helm template, and its like a crap shoot. They both inconsistently override the env variable. In Kubernetes' mind, it should override since the value is last. BUT we are using a helm template and then calling to yaml, which then I guess negates that. So strategy is back to the board. And this is the strategy for all the workers. I count 3 total repos needing this work.